modules/drivers/hive-like/deps.edn

```diff
@@ -5,7 +5,7 @@
  ;; implementations of things like log4j <-> slf4j, or are part of both hadoop-common and hive-jdbc;
  :deps
  {org.apache.thrift/libthrift {:mvn/version "0.19.0"}
-  org.apache.hive/hive-jdbc
+  org.apache.hive/hive-jdbc ^:antq/exclude ; update to 4 breaks everything
```

Comment:
Maybe some more context here? Is there anything we can link to?

Reply:
Only to my old closed PR which was an exercise in frustration. :)

I should've also collected all failures too and posted them in Slack, but was so frustrated I forgot about it.